---
content_type: page
description: This section contains lecture summaries for every week of the course.
learning_resource_types:
- Lecture Notes
ocw_type: CourseSection
title: Lecture Summaries
uid: 6a598894-7add-dfce-148d-4eccf9062f5f
---

{{< tableopen >}}
{{< theadopen >}}
{{< tropen >}}
{{< thopen >}}
WEEK #
{{< thclose >}}
{{< thopen >}}
TOPICS
{{< thclose >}}
{{< thopen >}}
LECTURE SUMMARIES
{{< thclose >}}

{{< trclose >}}

{{< theadclose >}}
{{< tropen >}}
{{< tdopen >}}
1
{{< tdclose >}}
{{< tdopen >}}
Introduction and Course Overview
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "week1" >}}{{< /anchor >}}Each instructor and student will begin by introducing him / herself to the class. In this first session we will discuss how to locate primary research publications. We will also discuss strategies for reading journal articles that will be useful during this course as well as in subsequent independent studies. In preparation for reading the assigned publications, the instructors will set the stage by providing a high level overview of the RNA field, including an introduction to basic cellular biology. We will also cover some of the laboratory techniques that will come up in the first few readings and introduce the topic of the next week.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
2
{{< tdclose >}}
{{< tdopen >}}
The discovery of miRNAs as gene regulators
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "week2" >}}{{< /anchor >}}We begin our exploration of the exciting world of non-coding RNAs with the discovery of microRNAs, a class of small non-coding RNAs that antagonize the function of complementary mRNAs. This work began largely with studies by Victor Ambros and Gary Ruvkun, who were interested in the regulation of the _lin-14_ gene in the nematode C. _elegans_. We will start by exploring the question these scientists were trying to address, which centered on trying to explain interesting features of developmental timing in worms, and follow their logic that led them to conclude a small regulatory RNA must play a pivotal role. We will then consider the discovery of the _let-7_ family of microRNAs, which because of its conservation and ubiquity in more complex organisms led to great interest amongst the scientific community in microRNAs.Techniques we will consider in understanding these papers are gel electrophoresis, hybridization and blotting, and genetics.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
3
{{< tdclose >}}
{{< tdopen >}}
Roles for miRNAs beyond gene downregulation
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "week3" >}}{{< /anchor >}}Having explored the discovery of microRNAs and their role in gene downregulation, we next turn to explore other possible functions of microRNAs. We will consider whether microRNAs serve only to turn off gene expression or whether they are used physiologically for additional purposes. We will start by considering the case in which microRNAs might help "buffer" a gene so that it is expressed in the needed range, and we will critically evaluate the author's claims about possible purposes and mechanism. Next we will read a paper that utilizes a single experimental system to thoroughly test a theoretical model about how microRNAs function in the reduction of gene expression noise. Students will explore how this theoretical treatment differs from the other papers we have considered this far, and together we will discuss the paper's strengths and weaknesses as a scientific approach. Techniques we will encounter and discuss include some introductory level concepts in population genetics, flow cytometry, theoretical modelling, and _Drosophila_ genetics.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
4
{{< tdclose >}}
{{< tdopen >}}
From siRNA discovery to therapeutic agent
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "week4" >}}{{< /anchor >}}In 1998, Andrew Fire and Craig Mello made an insightful observation that in the nematode C. _elegans_ one can interfere with the function of an endogenous gene by introducing double-stranded RNA molecules, a process referred to as RNA interference. In the years following this discovery, small interfering RNAs (siRNAs) became a widely used experimental tool for inhibiting endogenous gene expression. Moreover, clinical trials are underway for the delivery of siRNAS to tissues in the human body as a means of treating a variety of diseases. We will read about an early-stage clinical program in which in which siRNA therapy is being investigated as a potential therapy for anemia patients. In Querbes _et al_, siRNAs targetting an inhibitor of erythropoietin production are used to increase liver erythropoietin and improve red blood cell production in anemic mouse models. This class will illustrate how basic scientific studies of nematodes can incite novel disease therapies.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
5
{{< tdclose >}}
{{< tdopen >}}
Circular RNAs and the competing endogenous RNA hypothesis
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "week5" >}}{{< /anchor >}}If many small RNAs regulate their target through binding complementary sequence, what happens when they have more than one target that contains such a sequence? Will changes in the level of one target impact the other target? This idea is known as the competing endogenous RNA hypothesis. This week we consider this hypothesis amongst others. The emphasis will be on the line of thinking that led to the formulation of this hypothesis and what it might imply about the complexity of regulation by small non-coding RNAs. We first start by considering a type of non-coding RNA called circular RNAs (circRNAs) and whether these molecules can act in a regulatory fashion by binding other RNAs. Importantly, we will discuss next-generation sequencing technologies and how this approach has led to an explosion in our understanding of non-coding RNAs, including in this case the discovery of circRNA. Through discussion of this paper we will also consider the ways next-generation sequencing data can be processed and handled and how different processing techniques can generate different understanding of the experimental results. We next consider a case in which an expressed pseudogene might act to promote cancer in tumors by titrating away an important microRNA and diverting it from its target, in this case an important tumor suppressor gene. We will discuss and speculate together how such non protein-coding roles for transcripts that could potentially encode proteins might function in physiology.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
6
{{< tdclose >}}
{{< tdopen >}}
Role of lncRNAs in X chromosome inactivation
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "week6" >}}{{< /anchor >}}We have explored how small non-coding RNAs can regulate the expression of specific genes. We will begin a two-week study of long non-coding RNAs (lncRNAs), a general class of non-coding RNA greater than 200 base pairs. One of the best-characterized lncRNAs is involved in silencing the entire X chromosome. X inactivation is a process that occurs in mammalian females as a means of dosage compensation between males (XY) and females (XX). In 1922, Brown et al. characterized a gene, named Xist, which lacked protein-coding potential and was expressed exclusively from the inactive X chromosome. They hypothesized that Xist might function in X inactivation as a structural RNA molecule. A breakthrough in understanding how Xist silences the X chromosome came in 2008 when Zhao et al. showed that Xist recruits the polycomb repressive complex (PRC2) to the X chromosome. We will discuss how this lncRNA-protein interaction led to subsequent identification of many lncRNAs that interact with the PRC2 complex in the nucleus.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
7
{{< tdclose >}}
{{< tdopen >}}
Regulation of genomic imprinting by long non-coding RNAs
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "week7" >}}{{< /anchor >}}Long non-coding RNAs serve a wide variety of cellular functions in addition to X chromosome inactivation. In this class, we will discuss data that suggest lncRNAs might be involved in establishing imprinted gene expression. Genomic imprinting is an epigenetic process that results in the preferrential expression of a gene from one allele. This process is controlled by methylation at specific DNA sequences called imprint control regions, and lncRNAs might help mediate the effect of DNA methylation on expression of imprinted genes. In the first paper, Meng et al. decribe an imprinted lncRNA called Ube3a antisense transcript, Ube3a-ATS. By genetic insertion of a premature transcription termination cassette, they demonstrate that Ube3a-ATS is required for silencing the paternal copy of Ube3a in neurons. This discovery has direct disease relevance, because deletion of maternal Ube3a causes a neurodevelopmental disease called Angelman syndrome. In the second paper, Huang et al. performed a drug screen to identify small molecules that could unsilence paternal Ube3a. They identified a topoisomerase inhibitor that down-regulated Ube3a-ATS and up-regulated paternal Ube3a. We will discuss experiments to investigate the molecular mechanism by which Ube3a-ATS silences paternal Ube3a, and possible therapeutic strategies for Angelman syndrome.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
8
{{< tdclose >}}
{{< tdopen >}}
Field trip to Alnylam
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "week8" >}}{{< /anchor >}}No regular class scheduled this week.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
9
{{< tdclose >}}
{{< tdopen >}}
Catalytic functions of RNA and the RNA-world hypothesis: Ribozymes
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "week9" >}}{{< /anchor >}}For many years scientists thought, just as only DNA could truly carry information in a cell, only proteins could carry out enzymatic functions. Thus it was downright spectacular when Thomas Cech and Sidney Altman independently showed that RNA could also act as a catalyst with enzymatic properties, discoveries for which they later shared the Nobel Prize in Chemistry. This means that RNA can serve functions such as information coding like DNA and also enzymatic functions like protein. Since RNA can do it all, scientists reasoned, perhaps life began with multifunctional RNA instead of trying to evolve all the complicated macromolecular components of the cell at the same time. This idea is known as the RNA-world hypothesis, and we will discuss its origins following discussion of Thomas Cech's paper. Yet, viewing RNA as primarily driving the evolution of the early cell leads to some interesting challenges explaining all of the chemistry that must be involved. This week we will also discuss a paper that gives a potential explanation of how RNA could meet a key challenge in this early world. This paper will allow us to consider the technique of directed evolution and how it can be used to address scientific hypotheses. We will also briefly discuss some key concepts in chemistry as they apply to these papers such as catalysis, chirality, and RNA molecular structure.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
10
{{< tdclose >}}
{{< tdopen >}}
It's all about the ribosome
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "week10" >}}{{< /anchor >}}RNA as a key part of enzymatic machinery is more than just a dream in a test tube—RNA is the key component of ribosomes, the complexes that translate messages to proteins. Today we will continue along our theme of considering the catalytic properties of RNA. An early step in considering RNA as more than a simple intermediate molecule in the flow of genetic information came from research on ribosomal RNA (rRNA) that dates back to the 1950s. We will pick up the story in the early 1990s when Noller et al. use a simplified peptidyl transferase reaction to determine if RNA or protein components of the ribosome possessed catalytic activity. They argue that the peptidyl transferase activity of the ribosome was insensitive to protease treatment (aimed at degrading all protein in the sample) but abolished by ribonuclease treatment (aimed at degrading all RNA in the sample). rRNA is post-transcriptionally modified by another class of non-coding RNA called small nucleolar RNA (snoRNA). We will analyze the report from Kiss­ Laszlo et al. that showed through a series of elegant depletion and reconstitution experiments that snoRNAs directly base pair with pre-rRNA and this interaction is required for 2'-0-methylation at specific sites on pre-rRNA.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
11
{{< tdclose >}}
{{< tdopen >}}
Chemical modifications of mature RNA molecules
{{< tdclose >}}
{{< tdopen >}}


{{< anchor "week11" >}}{{< /anchor >}}Ribosomal RNA is not the only RNA that can be modified in a way that changes its meaning and function. In fact, recently there has been a veritable explosion in the discovery of chemical modifications that can be made to RNA molecules. We will examine one recent example of messenger RNA (mRNA) modification involving N{{< sup "6" >}}\-methyladenosine. We will discuss how these modifications were discovered as well as their potential role in the regulation of protein levels. We will speculate as to additional modifications that could be made to mRNAs and how we might go about discovering if such modifications occur. Then we will consider a case of RNA editing, whereby bases in an otherwise mature mRNA molecule can be replaced with entirely new bases. Such editing might have very important roles in the overall regulation of these messages, and we will consider if this is the case in Wilms tumor, a pediatric malignancy of the kidney. These papers will allow us to further our knowledge of next-generation sequencing techniques along with further considering how techniques from chemistry can help us understand the life sciences.

Note: At the end of class we will distribute a list of publications from which you will choose one for the final oral presentation. You may choose a paper not on the list with approval from the instructors.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
12
{{< tdclose >}}
{{< tdopen >}}
Harnessing an RNA-guided immune system to treat human disease
{{< tdclose >}}
{{< tdopen >}}


{{< anchor "week12" >}}{{< /anchor >}}During this class we will learn about a function of bacterial RNA that has recently driven a lot of excitement from scientists in a wide range of disciplines. Bacteria have evolved an adaptive immune system against invading viruses by incorporating pieces of viral DNA into their genome. This defense system is called clustered regularly interspaced short palindromic repeats (CRISPR). In this system, small pieces of viral DNA that are incorporated into the bacterial genome are transcribed into RNA. This RNA guides a CRISPR-associated (Cas) protein to bind invading viral DNA through sequence complementarity, ultimately resulting in cleavage and degradation of the viral DNA. We will read a paper from the laboratories of Jennifer Doudna and Emmanuelle Charpentier in which they made the insightful discovery that base pairing between two different RNA molecules, the CRISPR RNA (crRNA) and trans-activating crRNA (tracrRNA), recruits Cas9 to specific DNA sites. We will then talk about how scientists have exploited this bacterial defense system for the use of genome editing in mammalian cells. Schwank _et al_. use the CRISPR / Cas9 system to correct patient mutations in the cystic fibrosis transmembrane conductor receptor. We will balance the hype of CRISPR technology by discussing potential shortcomings and risks of the technology, as well as the controversy regarding the use of CRISPR to edit human embryos.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
13
{{< tdclose >}}
{{< tdopen >}}
Oral Presentations and Course Evaluations
{{< tdclose >}}
{{< tdopen >}}
{{< anchor "week13" >}}{{< /anchor >}}Refer to the {{% resource_link 69fff540-7cca-af54-a432-503f05b7328f "Assignments" %}} section regarding the requirements for the oral presentations. In addition, we will discuss the course in general, and course evaluations will be completed as a way for the students to provide feedback about their experience in the course.
{{< tdclose >}}

{{< trclose >}}

{{< tableclose >}}